.github/workflows/backend-deploy.yml

```diff
@@ -1,6 +1,11 @@
 name: Backend deployment
 
 on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'rms/**'
```

Comment:
We should also check for changes of pyproject.toml and the lockfile

Reply:
and Dockerfile as well

Reply:
added